src/features/devtools/DevToolsControls.tsx

```diff
@@ -34,9 +36,9 @@ export const DevToolsControls = () => {
       <Tabs.List className={classes.tabList}>
         <Tabs.Tab value={DevToolsTab.General}>{DevToolsTab.General}</Tabs.Tab>
         <Tabs.Tab value={DevToolsTab.Logs}>{DevToolsTab.Logs}</Tabs.Tab>
-        <Tabs.Tab value={DevToolsTab.Layout}>{DevToolsTab.Layout}</Tabs.Tab>
-        <Tabs.Tab value={DevToolsTab.Components}>{DevToolsTab.Components}</Tabs.Tab>
-        <Tabs.Tab value={DevToolsTab.Expressions}>{DevToolsTab.Expressions}</Tabs.Tab>
+        {isInForm && <Tabs.Tab value={DevToolsTab.Layout}>{DevToolsTab.Layout}</Tabs.Tab>}
+        {isInForm && <Tabs.Tab value={DevToolsTab.Components}>{DevToolsTab.Components}</Tabs.Tab>}
+        {isInForm && <Tabs.Tab value={DevToolsTab.Expressions}>{DevToolsTab.Expressions}</Tabs.Tab>}
```

Comment:
🤯

Reply:
Ahh, yes, forgot to mention that in the description, but DevTools should work outside of form-tasks now, with this change. It was very much needed when I added a custom error page when row IDs are not provided by the backend, and you need to run `altinn-studio` CLI to fix it - and the details about that should only be visible to developers, thus logging it at the same time as the (more generic) error page is shown was the best way I found to ensure that. Still, very useless when devtools crashed there.. :sweat_smile: